Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.7
+
+RUN apk add --no-cache python3 && \
+    python3 -m ensurepip && \
+    rm -r /usr/lib/python*/ensurepip && \
+    pip3 install --upgrade pip setuptools && \
+    if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi && \
+    if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3 /usr/bin/python; fi && \
+    rm -r /root/.cache
+
+
+RUN mkdir /battlesnakeio_play
```

Comment:
don't need to create the folder before using, copy will create for you. Also what do you think about just using a generic /app folder in the container.

Reply:
`/app` seems fine to me, probably more of a standard than project name, I will switch